src/scss/_responsive-flat.scss

```diff
@@ -22,7 +22,7 @@
 		padding: 8px;
 		width: 50%;
 
-		@include oGridRespondTo(S) {
+		@include oGridRespondTo(S) { // sass-lint:disable-line mixins-before-declarations
```

Comment:
this is going to be annoying to add to every instance we're using breakpoint mixins. It might be worth disabling this or seeing if we can ignore mixins with content in our sass-lint config?